### PR TITLE
CXTB-93 Review Participant

### PR DESCRIPTION
**Pull request by dguzmanneural**
-Add test
-Add page selectors
-Made the call from senior

### CODE REVIEW

File: tests/cases/care_partner_platform/case_care_platform_participants.yaml [hunk 1]
```diff
@@ -392,6 +385,218 @@ TestParticipantRecordListing:
           Var: SUBTAB_CLASS
           Value: 'active'
 
-
-
-
+#Care Partner Platform - Review Participant Summary
```
Comment (miguel-hquez) [inline tests/cases/care_partner_platform/case_care_platform_participants.yaml:388]: please add the test case number

File: tests/cases/care_partner_platform/case_care_platform_participants.yaml [hunk 2]
```diff
@@ -392,6 +385,218 @@ TestParticipantRecordListing:
...
+        - var: TAB_CLASS
+          attr: class
+          condition: nempty
+          match: "(.*)"
+    - Type: assert
+      Role: desktopChrome
+      Asserts:
+        - Type: contain
+          Var: TAB_CLASS
+          Value: 'active'
```
Comment (miguel-hquez) [inline tests/cases/care_partner_platform/case_care_platform_participants.yaml:602]: please add a tear down to hang up the call

File: tests/cases/care_partner_platform/case_care_platform_participants.yaml (hunk 1 above)
Comment (dguzmanneural) [reply tests/cases/care_partner_platform/case_care_platform_participants.yaml]: added

File: tests/cases/care_partner_platform/case_care_platform_participants.yaml (hunk 2 above)
Comment (dguzmanneural) [reply tests/cases/care_partner_platform/case_care_platform_participants.yaml]: added, even though is not included in the test, is the better finish for the environment and other tests

**Review by miguel-hquez — APPROVED** [review]

**Review by csanchez-nts — APPROVED** [review]

**Pull request merged**